contrib/localnet/scripts/start-zetaclientd.sh

```diff
@@ -84,63 +99,72 @@ MYIP=$(/sbin/ip -o -4 addr list eth0 | awk '{print $4}' | cut -d/ -f1)
     zetaclientd init --zetacore-url zetacore0 --chain-id athens_101-1 \
         --operator "$operatorAddress" --log-format=text --public-ip "$MYIP" \
         --keyring-backend "$BACKEND" --pre-params "$PREPARAMS_PATH"
 
-    # import relayer private key for zetaclient0
+    # Import relayer private key for zetaclient0.
     import_relayer_key 0
-    # add feature flags to existing config
+
+    # Add feature flags to existing config.
     enable_multiple_calls
 
-    # if eth2 is enabled, set the endpoint in the zetaclient_config.json
-    # in this case, the additional evm is represented with the sepolia chain, we set manually the eth2 endpoint to the sepolia chain (11155111 -> http://eth2:8545)
-    # in /root/.zetacored/config/zetaclient_config.json
+    # If eth2 is enabled, set the endpoint in the zetaclient_config.json.
+    # In this case, the additional evm is represented with the sepolia chain, and we manually set
+    # the eth2 endpoint to the sepolia chain (11155111 -> http://eth2:8545) in
+    # /root/.zetacored/config/zetaclient_config.json.
     if host eth2 > /dev/null; then
-     echo "enabling additional evm (eth2)"
-     set_sepolia_endpoint
+        echo "enabling additional evm (eth2)"
+        set_sepolia_endpoint
     fi
 fi
 
-if [[ $HOSTNAME != "zetaclient0" && ! -f ~/.zetacored/config/zetaclient_config.json ]] then
-  if [[ $HOSTNAME == "zetaclient-new-validator" ]]; then
-      num=1
-      node="zetacore-new-validator"
-  else
-      num=$(echo $HOSTNAME | tr -dc '0-9')
-      node="zetacore$num"
-  fi
-  
-  # TODO: use alternative DNS name (instead of IP) for other zetaclients (DNS should work as well)
-  # https://github.com/zeta-chain/node/issues/4374
-  zetaclientd init --zetacore-url "$node" --chain-id athens_101-1 \
-      --operator "$operatorAddress" --log-format=text --public-ip "$MYIP" \
-      --keyring-backend "$BACKEND" --pre-params "$PREPARAMS_PATH"
-
-  # import relayer private key for zetaclient{$num}
-  import_relayer_key "${num}"
-  # add feature flags to existing config
-  enable_multiple_calls
-
-  # check if the option is additional-evm
-  # in this case, the additional evm is represented with the sepolia chain, we set manually the eth2 endpoint to the sepolia chain (11155111 -> http://eth2:8545)
-  # in /root/.zetacored/config/zetaclient_config.json
-  if [[ -n $ADDITIONAL_EVM ]]; then
-   set_sepolia_endpoint
-  fi
+if [[ $HOSTNAME != "zetaclient0" && ! -f ~/.zetacored/config/zetaclient_config.json ]]; then
+    # Use alternative DNS name (instead of IP) for other zetaclients (DNS should work as well).
+    # See: https://github.com/zeta-chain/node/issues/4374.
+    zetaclientd init --zetacore-url "$node" --chain-id athens_101-1 \
+        --operator "$operatorAddress" --log-format=text --public-ip "$MYIP" \
+        --keyring-backend "$BACKEND" --pre-params "$PREPARAMS_PATH"
+
+    # Import relayer private key for zetaclient{$num}.
+    if [[ $HOSTNAME != "zetaclient-dry" ]]; then
```

Comment:
can we add comment here as well? also should we enable_multiple_calls if zetaclient-dry?